Code/Framework/AzToolsFramework/AzToolsFramework/Metadata/MetadataManager.cpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/Metadata/MetadataManager.h>
+
+namespace AzToolsFramework
+{
+    void MetadataManager::Reflect(AZ::ReflectContext* context)
+    {
+        if (auto* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<MetadataManager>()->Version(1);
```

Comment:
Should this version be set to MetadataVersion? Is there a reason you wouldn't want them in sync?

Reply:
Interesting thought, I think maybe they should be separate though, because technically they're different serializations.  MetadataVersion is the version for the file format, this is the version for the component itself, which doesn't have any data (and probably won't...).

Reply:
That makes sense, maybe add a comment in the code itself, too, so someone else doesn't think the same thing later?